src/arch/netlrts/conv-common.h

```diff
@@ -23,7 +23,7 @@
 */
 #define CMK_MSG_HEADER_BASIC   CMK_MSG_HEADER_EXT
 
-#define CMK_MSG_HEADER_EXT_    CmiUInt2 d0,d1,d2,d3,hdl,type,xhdl,info,redID,rank; CmiInt4 root, size; CmiUInt1 zcMsgType:4, cmaMsgType:2, nokeep:1;
+#define CMK_MSG_HEADER_EXT_     CmiInt4 root, size; CmiUInt2 d0,d1,d2,d3,hdl,type,xhdl,info,redID,rank; CmiUInt1 zcMsgType:4, cmaMsgType:2, nokeep:1;
```

Comment:
As pointed out in the comment above CMK_MSG_HEADER_BASIC machine-dgram.C must be kept consistent with the ordering of members here, so that code will need to be updated as well.

Reply:
I changed the order back to the original since there anyway isn't any alignment issue here